<launch>
   <param name="use_sim_time" value="true"/>

   <node name="lidar_undistortion_2d_node" pkg="lidar_undistortion_2d" type="lidar_undistortion_2d_node">
      <param name="scan_sub_topic" value="/scan"/>
      <param name="scan_pub_topic" value="/scan_undistortion"/>
      <param name="enable_pub_pointcloud" value="true"/>
      <param name="pointcloud_pub_topic" value="/pointcloud_undistortion"/>
      <param name="lidar_frame" value="laser_link"/>
      <param name="odom_frame" value="odom"/>
      <param name="lidar_scan_time_gain" value="1.0"/>
   </node>

   <node pkg="rviz" type="rviz" name="rviz" args="-d $(find lidar_undistortion)/launch/racecar_full.rviz"/>

</launch>
